$MirOS: ports/misc/screen/patches/patch-doc_screen_1,v 1.4 2007/02/11 01:33:45 tg Exp $
$OpenBSD: patch-doc_screen_1,v 1.4 2003/09/26 23:50:55 brad Exp $
--- doc/screen.1.orig	Fri Dec  5 13:51:35 2003
+++ doc/screen.1	Tue Nov 28 20:41:25 2017
@@ -64,28 +64,19 @@ exits.
 Everything you type is sent to the program running in the current window.
 The only exception to this is the one keystroke that is used to initiate
 a command to the window manager.
-By default, each command begins with a control-a (abbreviated C-a from
+By default, each command begins with a control-a (abbreviated ^A from
 now on), and is followed by one other keystroke.
 The command character and all the key bindings can be fully customized
 to be anything you like, though they are always two characters in length.
 .PP
-.I Screen
-does not understand the prefix \*QC-\*U to mean control. 
-Please use the caret notation (\*Q^A\*U instead of \*QC-a\*U) as arguments
-to e.g. the 
-.I escape
-command or the \fI-e\fP option.
-.I Screen
-will also print out control characters in caret notation.
-.PP
-The standard way to create a new window is to type \*QC-a c\*U.
+The standard way to create a new window is to type \*Q^Ac\*U.
 This creates a new window running a shell and switches to that
 window immediately, regardless of the state of the process running
 in the current window.
 Similarly, you can create a new window with a custom command in it by
 first binding the command to a keystroke (in your .screenrc file or at the
-\*QC-a :\*U command line) and
-then using it just like the \*QC-a c\*U command.
+\*Q^A:\*U command line) and
+then using it just like the \*Q^Ac\*U command.
 In addition, new windows can be created by running a command like:
 .IP
 screen emacs prog.c
@@ -99,7 +90,7 @@ create the new window.
 The above example would start the emacs editor (editing prog.c) and switch
 to its window.
 .PP
-If \*Q/etc/utmp\*U is writable by
+If \*Q/var/run/utmp\*U is writable by
 .IR screen ,
 an appropriate record will be written to this file for each window, and
 removed when the window is terminated.
@@ -108,7 +99,7 @@ This is useful for working with \*Qtalk\
 file to determine who you are. As long as
 .I screen
 is active on your terminal,
-the terminal's own record is removed from the utmp file. See also \*QC-a L\*U.
+the terminal's own record is removed from the utmp file. See also \*Q^AL\*U.
 
 
 .SH GETTING STARTED
@@ -121,7 +112,7 @@ just as you would for any other termcap/
 for example.)
 .PP
 If you're impatient and want to get started without doing a lot more reading,
-you should remember this one command:  \*QC-a ?\*U.
+you should remember this one command:  \*Q^A?\*U.
 Typing these two characters will display a list of the available
 .I screen
 commands and their bindings. Each keystroke is discussed in
@@ -170,7 +161,7 @@ does not start
 .IR screen ,
 but detaches the elsewhere running
 .I screen
-session. It has the same effect as typing \*QC-a d\*U from
+session. It has the same effect as typing \*Q^Ad\*U from
 .IR screen 's
 controlling terminal. \fB\-D\fP is the equivalent to the power detach key.
 If no session can be detached, this option is ignored. In combination with the
@@ -204,7 +195,7 @@ Note: It is always a good idea to check 
 .BI "\-e " xy
 specifies the command character to be \fIx\fP and the character generating a
 literal command character to \fIy\fP (when typed after the command character).
-The default is \*QC-a\*U and `a', which can be specified as \*Q-e^Aa\*U.
+The default is \*Q^A\*U and 'a', which can be specified as \*Q-e^Aa\*U.
 When creating a
 .I screen
 session, this option sets the default command character. In a multiuser
@@ -222,13 +213,13 @@ This can also be defined through the \*Q
 Specifies the history scrollback buffer to be \fInum\fP lines high.
 .TP 5
 .B \-i
-will cause the interrupt key (usually C-c) to interrupt the display
+will cause the interrupt key (usually ^C) to interrupt the display
 immediately when flow-control is on.
 See the \*Qdefflow\*U .screenrc command for details.
 The use of this option is discouraged.
 .TP 5
 .BR \-l " and " \-ln
-turns login mode on or off (for /etc/utmp updating).
+turns login mode on or off (for /var/run/utmp updating).
 This can also be defined through the \*Qdeflogin\*U .screenrc command.
 .TP 5
 .BR \-ls " and " \-list
@@ -373,215 +364,219 @@ work if the session is password protecte
 As mentioned, each
 .I screen
 command consists of a
-\*QC-a\*U followed by one other character.
+\*Q^A\*U followed by one other character.
 For your convenience, all commands that are bound to lower-case letters are
 also bound to their control character counterparts (with the exception
-of \*QC-a a\*U; see below), thus, \*QC-a c\*U as well as \*QC-a C-c\*U can
+of \*Q^Aa\*U; see below), thus, \*Q^Ac\*U as well as \*Q^A^C\*U can
 be used to create a window. See section \*QCUSTOMIZATION\*U for a description
 of the command.
 .PP
-.TP 26n
+.TP 28n
 The following table shows the default key bindings:
-.IP "\fBC-a '\fP	(select)"
+.IP "\fB^A'\fP	(select)"
 Prompt for a window name or number to switch to.
-.IP "\fBC-a ""\fP	(windowlist -b)"
+.IP "\fB^A""\fP	(windowlist -b)"
 Present a list of all windows for selection.
-.IP "\fBC-a 0\fP	(select 0)"
+.IP "\fB^A0\fP	(select 0)"
 .PD 0
 .IP "\fB ... \fP	   ..."
-.IP "\fBC-a 9\fP	(select 9)"
-.IP "\fBC-a -\fP	(select -)"
+.IP "\fB^A9\fP	(select 9)"
+.IP "\fB^A-\fP	(select -)"
 .PD
 Switch to window number 0 \- 9, or to the blank window.
-.IP "\fBC-a tab\fP	(focus)"
+.IP "\fB^A<tab>\fP	(focus)"
 .PD
 Switch the input focus to the next region.
-.IP "\fBC-a C-a\fP	(other)"
+.IP "\fB^A^A\fP	(other)"
 Toggle to the window displayed previously.
 Note that this binding defaults to the command character typed twice,
 unless overridden.  For instance, if you use the option \*Q\fB\-e]x\fP\*U,
 this command becomes \*Q]]\*U.
-.IP "\fBC-a a\fP	(meta)"
-Send the command character (C-a) to window. See \fIescape\fP command.
-.IP "\fBC-a A\fP	(title)"
+.IP "\fB^Aa\fP	(meta)"
+Send the command character (^A) to window. See \fIescape\fP command.
+.IP "\fB^AA\fP	(title)"
 Allow the user to enter a name for the current window.
-.IP "\fBC-a b\fP"
+.IP "\fB^Ab\fP"
 .PD 0
-.IP "\fBC-a C-b\fP	(break)"
+.IP "\fB^A^B\fP	(break)"
 .PD
 Send a break to window.
-.IP "\fBC-a B\fP	(pow_break)"
+.IP "\fB^AB\fP	(pow_break)"
 Reopen the terminal line and send a break.
-.IP "\fBC-a c\fP"
+.IP "\fB^Ac\fP"
 .PD 0
-.IP "\fBC-a C-c\fP	(screen)"
+.IP "\fB^A^C\fP	(screen)"
 .PD
 Create a new window with a shell and switch to that window.
-.IP "\fBC-a C\fP	(clear)"
+.IP "\fB^AC\fP	(clear)"
 Clear the screen.
-.IP "\fBC-a d\fP"
+.IP "\fB^Ad\fP"
 .PD 0
-.IP "\fBC-a C-d\fP	(detach)"
+.IP "\fB^A^D\fP	(detach)"
 .PD
 Detach
 .I screen
 from this terminal.
-.IP "\fBC-a D D\fP	(pow_detach)"
+.IP "\fB^AD D\fP	(pow_detach)"
 Detach and logout.
-.IP "\fBC-a f\fP"
+.IP "\fB^Af\fP"
 .PD 0
-.IP "\fBC-a C-f\fP	(flow)"
+.IP "\fB^A^F\fP	(flow)"
 .PD
 Toggle flow \fIon\fP, \fIoff\fP or \fIauto\fP.
-.IP "\fBC-a F\fP	(fit)"
+.IP "\fB^AF\fP	(fit)"
 Resize the window to the current region size.
-.IP "\fBC-a C-g\fP	(vbell)"
+.IP "\fB^A^G\fP	(vbell)"
 Toggles
 .I screen's
 visual bell mode.
-.IP "\fBC-a h\fP	(hardcopy)"
+.IP "\fB^Ah\fP	(hardcopy)"
 .PD
 Write a hardcopy of the current window to the file \*Qhardcopy.\fIn\fP\*U.
-.IP "\fBC-a H\fP	(log)"
+.IP "\fB^AH\fP	(log)"
 Begins/ends logging of the current window to the file \*Qscreenlog.\fIn\fP\*U.
-.IP "\fBC-a i\fP"
+.IP "\fB^Ai\fP"
 .PD 0
-.IP "\fBC-a C-i\fP	(info)"
+.IP "\fB^A^I\fP	(info)"
 .PD
 Show info about this window.
-.IP "\fBC-a k\fP"
+.IP "\fB^Ak\fP"
 .PD 0
-.IP "\fBC-a C-k\fP	(kill)"
+.IP "\fB^A^K\fP	(kill)"
 .PD
 Destroy current window.
-.IP "\fBC-a l\fP"
+.IP "\fB^Al\fP"
 .PD 0
-.IP "\fBC-a C-l\fP	(redisplay)"
+.IP "\fB^A^L\fP	(redisplay)"
 .PD
 Fully refresh current window.
-.IP "\fBC-a L\fP	(login)"
+.IP "\fB^AL\fP	(login)"
 Toggle this windows login slot. Available only if
 .I screen
 is configured to update the utmp database.
-.IP "\fBC-a m\fP"
+.IP "\fB^Am\fP"
 .PD 0
-.IP "\fBC-a C-m\fP	(lastmsg)"
+.IP "\fB^A^M\fP	(lastmsg)"
 .PD
 Repeat the last message displayed in the message line.
-.IP "\fBC-a M\fP	(monitor)"
+.IP "\fB^AM\fP	(monitor)"
 Toggles monitoring of the current window.
-.IP "\fBC-a space\fP"
+.IP "\fB^A<space>\fP"
 .PD 0
-.IP "\fBC-a n\fP"
-.IP "\fBC-a C-n\fP	(next)"
+.IP "\fB^An\fP"
+.IP "\fB^A^N\fP	(next)"
 .PD
 Switch to the next window.
-.IP "\fBC-a N\fP	(number)"
+.IP "\fB^AN\fP	(number)"
 Show the number (and title) of the current window.
-.IP "\fBC-a backspace\fP"
+.IP "\fB^A<backspace>\fP"
 .PD 0
-.IP "\fBC-a h\fP"
-.IP "\fBC-a p\fP"
-.IP "\fBC-a C-p\fP	(prev)"
+.IP "\fB^Ah\fP"
+.IP "\fB^Ap\fP"
+.IP "\fB^A^P\fP	(prev)"
 .PD
-Switch to the previous window (opposite of \fBC-a n\fP).
-.IP "\fBC-a q\fP"
+Switch to the previous window (opposite of \fB^An\fP).
+.IP "\fB^Aq\fP"
 .PD 0
-.IP "\fBC-a C-q\fP	(xon)"
+.IP "\fB^A^Q\fP	(xon)"
 .PD
 Send a control-q to the current window.
-.IP "\fBC-a Q\fP	(only)"
+.IP "\fB^AQ\fP	(only)"
 Delete all regions but the current one.
-.IP "\fBC-a r\fP"
+.IP "\fB^Ar\fP"
 .PD 0
-.IP "\fBC-a C-r\fP	(wrap)"
+.IP "\fB^A^R\fP	(wrap)"
 .PD
 Toggle the current window's line-wrap setting (turn the current window's
 automatic margins on and off).
-.IP "\fBC-a s\fP"
+.IP "\fB^As\fP"
 .PD 0
-.IP "\fBC-a C-s\fP	(xoff)"
+.IP "\fB^A^S\fP	(xoff)"
 .PD
 Send a control-s to the current window.
-.IP "\fBC-a S\fP	(split)"
+.IP "\fB^AS\fP	(split)"
 Split the current region into two new ones.
-.IP "\fBC-a t\fP"
+.IP "\fB^At\fP"
 .PD 0
-.IP "\fBC-a C-t\fP	(time)"
+.IP "\fB^A^T\fP	(time)"
 .PD
 Show system information.
-.IP "\fBC-a v\fP	(version)"
+.IP "\fB^Au\fP	(digraph '\`?')"
 .PD
+Enter hexadecimal Unicode codepoint (in UTF-8 mode)
+or octet (in 8-bit mode).
+.IP "\fB^Av\fP	(version)"
+.PD
 Display the version and compilation date.
-.IP "\fBC-a C-v\fP	(digraph)"
+.IP "\fB^A^V\fP	(digraph)"
 .PD
 Enter digraph.
-.IP "\fBC-a w\fP"
+.IP "\fB^Aw\fP"
 .PD 0
-.IP "\fBC-a C-w\fP	(windows)"
+.IP "\fB^A^W\fP	(windows)"
 .PD
 Show a list of window.
-.IP "\fBC-a W\fP	(width)"
+.IP "\fB^AW\fP	(width)"
 Toggle 80/132 columns.
-.IP "\fBC-a x\fP"
+.IP "\fB^Ax\fP"
 .PD 0
-.IP "\fBC-a C-x\fP	(lockscreen)"
+.IP "\fB^A^X\fP	(lockscreen)"
 .PD
 Lock this terminal.
-.IP "\fBC-a X\fP 	(remove)"
+.IP "\fB^AX\fP 	(remove)"
 Kill the current region.
-.IP "\fBC-a z\fP"
+.IP "\fB^Az\fP"
 .PD 0
-.IP "\fBC-a C-z\fP	(suspend)"
+.IP "\fB^A^Z\fP	(suspend)"
 .PD
 Suspend
 .IR screen .
 Your system must support BSD-style job-control.
-.IP "\fBC-a Z\fP	(reset)"
+.IP "\fB^AZ\fP	(reset)"
 Reset the virtual terminal to its \*Qpower-on\*U values.
-.IP "\fBC-a .\fP	(dumptermcap)"
+.IP "\fB^A.\fP	(dumptermcap)"
 Write out a \*Q.termcap\*U file.
-.IP "\fBC-a ?\fP	(help)"
+.IP "\fB^A?\fP	(help)"
 Show key bindings.
-.IP "\fBC-a C-\e\fP	(quit)"
+.IP "\fB^A^\e\fP	(quit)"
 Kill all windows and terminate
 .IR screen .
-.IP "\fBC-a :\fP	(colon)"
+.IP "\fB^A:\fP	(colon)"
 Enter command line mode.
-.IP "\fBC-a [\fP"
+.IP "\fB^A[\fP"
 .PD 0
-.IP "\fBC-a C-[\fP"
-.IP "\fBC-a esc\fP	(copy)"
+.IP "\fB^A^[\fP"
+.IP "\fB^Aesc\fP	(copy)"
 .PD
 Enter copy/scrollback mode.
-.IP "\fBC-a ]\fP	(paste .)"
+.IP "\fB^A]\fP	(paste .)"
 .PD
 Write the contents of the paste buffer to the stdin queue of the
 current window. 
-.IP "\fBC-a {\fP
+.IP "\fB^A{\fP
 .PD 0
-.IP "\fBC-a }\fP	(history)"
+.IP "\fB^A}\fP	(history)"
 .PD
 Copy and paste a previous (command) line.
-.IP "\fBC-a >\fP	(writebuf)"
-Write paste buffer to a file.
-.IP "\fBC-a <\fP	(readbuf)"
-Reads the screen-exchange file into the paste buffer.
-.IP "\fBC-a =\fP	(removebuf)"
-Removes the file used by \fBC-a <\fP and \fPC-a >\fP.
-.IP "\fBC-a ,\fP	(license)"
+.IP "\fB^A<\fP	(bumpleft)"
+Moves the current window left.
+.IP "\fB^A>\fP	(bumpright)"
+Moves the current window right.
+.IP "\fB^A=\fP	(collapse)"
+Reorders window positions to remove gaps in window numbers.
+.IP "\fB^A,\fP	(license)"
 Shows where
 .I screen
 comes from, where it went to and why you can use it.
-.IP "\fBC-a _\fP	(silence)"
+.IP "\fB^A_\fP	(silence)"
 Start/stop monitoring the current window for inactivity.
-.IP "\fBC-a *\fP	(displays)"
+.IP "\fB^A*\fP	(displays)"
 Show a listing of all currently attached displays.
 
 
 .SH CUSTOMIZATION
 The \*Qsocket directory\*U defaults either to $HOME/.screen or simply to
-/tmp/screens or preferably to /usr/local/screens chosen at compile-time. If
+/tmp/screens chosen at compile-time. If
 .I screen
 is installed setuid-root, then the administrator
 should compile
@@ -594,7 +589,7 @@ in the environment variable $SCREENDIR.
 When
 .I screen
 is invoked, it executes initialization commands from the files
-\*Q/usr/local/etc/screenrc\*U and
+\*Q@@SYSCONFDIR@@/screenrc\*U and
 \*Q.screenrc\*U in the user's home directory. These are the \*Qprogrammer's
 defaults\*U that can be overridden in the following ways: for the
 global screenrc file 
@@ -623,12 +618,12 @@ versions, as now the '$'-character has t
 variable substitution shall be performed. A string in single-quotes is also
 protected from variable substitution.
 .PP
-Two configuration files are shipped as examples with your screen distribution:
-\*Qetc/screenrc\*U and \*Qetc/etcscreenrc\*U. They contain a number of
+The file \*Q@@SYSCONFDIR@@/screenrc\*U is already pre-loaded with
+a number of
 useful examples for various commands.
 .PP
 Customization can also be done 'on-line'. To enter the command mode type
-`C-a :'. Note that commands starting with \*Qdef\*U change default values,
+'^A:'. Note that commands starting with \*Qdef\*U change default values,
 while others change current settings.
 .PP
 The following commands are available:
@@ -740,7 +735,7 @@ The default message is
 	'Activity in window %n'
 .sp
 Note that monitoring is off for all windows by default, but can be altered
-by use of the \*Qmonitor\*U command (C-a M).
+by use of the \*Qmonitor\*U command (^AM).
 .sp
 .ne 3
 .BR "allpartial on" | off
@@ -913,11 +908,11 @@ By default, most of the commands provide
 .I screen
 are bound to one or more keys as indicated in the \*QDEFAULT KEY BINDINGS\*U
 section, e.\|g. the
-command to create a new window is bound to \*QC-c\*U and \*Qc\*U.
+command to create a new window is bound to \*Q^C\*U and \*Qc\*U.
 The \*Qbind\*U command can be used to redefine the key bindings and to
 define new bindings.
 The \fIkey\fP argument is either a single character, a two-character sequence
-of the form \*Q^x\*U (meaning \*QC-x\*U), a backslash followed by an octal
+of the form \*Q^X\*U, a backslash followed by an octal
 number (specifying the ASCII code of the character), or a backslash followed
 by a second character, such as \*Q\e^\*U or \*Q\e\e\*U.
 The argument can also be quoted, if you like.
@@ -942,11 +937,11 @@ Some examples:
 .fi
 .PP
 would bind the space key to the command that displays a list
-of windows (so that the command usually invoked by \*QC-a C-w\*U
-would also be available as \*QC-a space\*U). The next three lines
-remove the default kill binding from \*QC-a C-k\*U and \*QC-a k\*U. 
-\*QC-a K\*U is then bound to the kill command. Then it
-binds \*QC-f\*U to the command \*Qcreate a window with a TELNET
+of windows (so that the command usually invoked by \*Q^A^W\*U
+would also be available as \*Q^Aspace\*U). The next three lines
+remove the default kill binding from \*Q^A^K\*U and \*Q^Ak\*U. 
+\*Q^AK\*U is then bound to the kill command. Then it
+binds \*Q^F\*U to the command \*Qcreate a window with a TELNET
 connection to foobar\*U, and bind \*Qescape\*U to the command
 that creates an non-login window with a.\|k.\|a. \*Qroot\*U in slot #9, with
 a superuser shell and a scrollback buffer of 1000 lines.
@@ -958,7 +953,7 @@ a superuser shell and a scrollback buffe
 	bindkey "^B" command -c demo1
 .fi
 .PP
-makes \*QC-b 0\*U select window 10, \*QC-b 1\*U window 11, etc.
+makes \*Q^B0\*U select window 10, \*Q^B1\*U window 11, etc.
 .PP
 .nf
 	bind -c demo2 0 select 10
@@ -967,7 +962,7 @@ makes \*QC-b 0\*U select window 10, \*QC
 	bind - command -c demo2
 .fi
 .PP
-makes \*QC-a - 0\*U select window 10, \*QC-a - 1\*U window 11, etc.
+makes \*Q^A- 0\*U select window 10, \*Q^A- 1\*U window 11, etc.
 .sp
 .ne 3
 .B bindkey
@@ -1105,12 +1100,22 @@ the
 window (using the paste buffer, where a copy remains):
 .PP
 .nf
-	C-a : bufferfile /etc/passwd
-	C-a < C-a ]
-	C-a : bufferfile
+	^A: bufferfile /etc/passwd
+	^A< ^A]
+	^A: bufferfile
 .fi
 .sp
 .ne 3
+.B bumpleft
+.PP
+Swaps window with previous one on window list.
+.sp
+.ne 3
+.B bumpright
+.PP
+Swaps window with next one on window list.
+.sp
+.ne 3
 .BR "c1 " [ on | off ]
 .PP
 Change c1 code processing. \*QC1 on\*U tells screen to treat
@@ -1167,8 +1172,8 @@ Change the \fIcurrent directory\fP of
 to the specified directory or, if called without an argument,
 to your home directory (the value of the environment variable $HOME).
 All windows that are created by means of the \*Qscreen\*U command
-from within \*Q.screenrc\*U or by means of \*QC-a : screen ...\*U
-or \*QC-a c\*U use this as their default directory.
+from within \*Q.screenrc\*U or by means of \*Q^A: screen ...\*U
+or \*Q^Ac\*U use this as their default directory.
 Without a chdir command, this would be the directory from which
 .I screen
 was invoked.
@@ -1185,6 +1190,11 @@ affect all the windows you create intera
 Clears the current window and saves its image to the scrollback buffer.
 .sp
 .ne 3
+.B collapse
+.PP
+Reorders windows on window list, removing number gaps between them.
+.sp
+.ne 3
 .B colon
 .RI [ prefix ]
 .PP
@@ -1197,7 +1207,7 @@ starting with 'def...'. 
 
 If you consider this as the `Ex command mode' of 
 .IR screen ,
-you may regard \*QC-a esc\*U (copy mode) as its `Vi command mode'.
+you may regard \*Q^Aesc\*U (copy mode) as its 'Vi command mode'.
 .sp 
 .ne 3
 .B command
@@ -1261,11 +1271,11 @@ of the top, center or bottom line of the
 \fBB\fP, \fBE\fP move the cursor WORD by WORD (as in vi).
 .br 
 .ti -2n
-\fBC-u\fP and \fBC-d\fP scroll the display up/down by the specified amount of 
+\fB^U\fP and \fB^D\fP scroll the display up/down by the specified amount of 
 lines while preserving the cursor position. (Default: half screen-full). 
 .br
 .ti -2n
-\fBC-b\fP and \fBC-f\fP scroll the display up/down a full screen.
+\fB^B\fP and \fB^F\fP scroll the display up/down a full screen.
 .br
 .ti -2n
 \fBg\fP moves to the beginning of the buffer.
@@ -1309,7 +1319,7 @@ digits 
 \fB0\fP..\fB9\fP which
 is taken as a repeat count. 
 .br
-Example: \*QC-a C-[ H 10 j 5 Y\*U will copy lines
+Example: \*Q^A^[ H 10 j 5 Y\*U will copy lines
 11 to 15 into the paste buffer.
 .br
 .ti -4n
@@ -1319,9 +1329,9 @@ Example: \*QC-a C-[ H 10 j 5 Y\*U will c
 .ti -2n
 \fB?\fP \fIVi\fP-like search backward.
 .ti -2n 
-\fBC-a s\fP \fIEmacs\fP style incremental search forward.
+\fB^As\fP \fIEmacs\fP style incremental search forward.
 .ti -2n
-\fBC-r\fP \fIEmacs\fP style reverse i-search.
+\fB^R\fP \fIEmacs\fP style reverse i-search.
 .ti -4n
 .IR Specials :
 .br
@@ -1337,14 +1347,14 @@ does. Press 
 given, both default to the current cursor position. 
 .br
 Example: Try this on a rather full text screen: 
-\*QC-a [ M 20 l SPACE c 10 l 5 j C SPACE\*U.
+\*Q^A[ M 20 l SPACE c 10 l 5 j C SPACE\*U.
 
 This moves one to the middle line of the screen, moves in 20 columns left,
 marks the beginning of the paste buffer, sets the left column, moves 5 columns
 down, sets the right column, and then marks the end of
 the paste buffer. Now try:
 .br
-\*QC-a [ M 20 l SPACE 10 l 5 j SPACE\*U
+\*Q^A[ M 20 l SPACE 10 l 5 j SPACE\*U
 
 and notice the difference in the amount of text copied.
 .br
@@ -1373,10 +1383,10 @@ the screen-exchange file (/tmp/screen-ex
 finished. 
 .br
 This example demonstrates how to dump the whole scrollback buffer 
-to that file: \*QC-A [ g SPACE G $ >\*U.
+to that file: \*Q^A[ g SPACE G $ >\*U.
 .br
 .ti -2n
-\fBC-g\fP gives information about the current line and column.
+\fB^G\fP gives information about the current line and column.
 .br
 .ti -2n
 \fBx\fP exchanges the first mark and the current cursor position. You
@@ -1398,7 +1408,7 @@ No longer exists, use \*Qreadreg\*U inst
 .ne 3
 .BR "crlf " [ on | off ]
 .PP
-This affects the copying of text regions with the `C-a [' command. If it is set
+This affects the copying of text regions with the '^A[' command. If it is set
 to `on', lines will be separated by the two character sequence `CR' - `LF'. 
 Otherwise (default) only `LF' is used.
 When no parameter is given, the state is toggled.
@@ -1583,7 +1593,7 @@ windows is changed. Initial setting is `
 .PP
 Same as the \fBwrap\fP command except that the default setting for new 
 windows is changed. Initially line-wrap is on and can be toggled with the 
-\*Qwrap\*U command (\*QC-a r\*U) or by means of "C-a : wrap on|off".
+\*Qwrap\*U command (\*Q^Ar\*U) or by means of "^A: wrap on|off".
 .sp
 .ne 3
 .BR "defwritelock on" | off | auto
@@ -1669,7 +1679,7 @@ and then compile the entry with 
 The echo command may be used to annoy 
 .I screen
 users with a 'message of the
-day'. Typically installed in a global /local/etc/screenrc. 
+day'. Typically installed in a global @@SYSCONFDIR@@/screenrc. 
 The option \*Q-n\*U may be used to suppress the line feed.
 See also \*Qsleep\*U.
 Echo is also useful for online checking of environment variables.
@@ -1684,7 +1694,9 @@ how to interpret the input/output. The f
 sets the encoding of the current window. Each window can emulate
 a different encoding. The optional second parameter overwrites
 the encoding of the connected terminal. It should never be
-needed as screen uses the locale setting to detect the encoding.
+needed as screen uses the locale setting
+as well as a special mechanism to determine a UTF-8 or ISO-8859-1 terminal
+to detect the encoding.
 There is also a way to select a terminal encoding depending on
 the terminal type by using the \*QKJ\*U termcap entry.
 
@@ -1702,7 +1714,7 @@ Set the command character to \fIx\fP and
 command character (by triggering the \*Qmeta\*U command) to \fIy\fP (similar
 to the \-e option).
 Each argument is either a single character, a two-character sequence
-of the form \*Q^x\*U (meaning \*QC-x\*U), a backslash followed by an octal
+of the form \*Q^X\*U, a backslash followed by an octal
 number (specifying the ASCII code of the character), or a backslash followed
 by a second character, such as \*Q\e^\*U or \*Q\e\e\*U.
 The default is \*Q^Aa\*U.
@@ -1864,7 +1876,7 @@ scrollback buffer.
 .PP
 If set to "on", 
 .I screen
-will append to the "hardcopy.n" files created by the command \*QC-a h\*U, 
+will append to the "hardcopy.n" files created by the command \*Q^Ah\*U, 
 otherwise these files are overwritten each time.
 Default is `off'.
 .sp
@@ -1965,7 +1977,7 @@ For example csh has the command \*Q!!\*U
 .I Screen
 allows you to have a primitive way of re-calling \*Qthe command that
 started ...\*U: You just type the first letter of that command, then hit
-`C-a {' and
+'^A{' and
 .I screen
 tries to find a previous line that matches with the `prompt character' 
 to the left of the cursor. This line is pasted into this window's input queue.
@@ -2056,9 +2068,9 @@ switches to the previously displayed win
 Note:
 .I Emacs
 users should keep this command in mind, when killing a line.
-It is recommended not to use \*QC-a\*U as the
+It is recommended not to use \*Q^A\*U as the
 .I screen
-escape key or to rebind kill to \*QC-a K\*U.
+escape key or to rebind kill to \*Q^AK\*U.
 .sp
 .ne 3
 .B lastmsg
@@ -2125,7 +2137,7 @@ This controls if the window is `logged i
 When no parameter is given, the login state of the window is toggled.
 Additionally to that toggle, it is convenient having a `log in' and a `log out'
 key. E.\|g. `bind I login on' and `bind O login off' will map these
-keys to be C-a I and C-a O.
+keys to be ^AI and ^AO.
 The default setting (in config.h.in) should be \*Qon\*U for a 
 .I screen
 that runs under suid-root.
@@ -2187,7 +2199,7 @@ See also \*Qbindkey\*U.
 This is a method of changing the keymap used for copy/history mode.
 The string is made up of \fIoldchar\fP=\fInewchar\fP pairs which are
 separated by `:'. Example: The string \*QB=^B:F=^F\*U will change the 
-keys `C-b' and `C-f' to the vi style binding (scroll up/down fill page).
+keys '^B' and '^F' to the vi style binding (scroll up/down fill page).
 This happens to be the default binding for `B' and `F'.
 The command \*Qmarkkeys h=^B:l=^F:$=^E\*U would set the mode for an emacs-style
 binding.
@@ -2207,7 +2219,7 @@ already existing windows. The number may
 .ne 3
 .B meta
 .PP
-Insert the command character (C-a) in the current window's input stream.
+Insert the command character (^A) in the current window's input stream.
 .sp
 .ne 3
 .BR "monitor " [ on | off ]
@@ -2436,9 +2448,9 @@ keyboard. This command can be used to bi
 .PP
 Kill all windows and terminate
 .IR screen .
-Note that on VT100-style terminals the keys C-4 and C-\e are identical.
+Note that on VT100-style terminals the keys ^4 and ^\e are identical.
 This makes the default bindings dangerous:
-Be careful not to type C-a C-4 when selecting window no. 4.
+Be careful not to type ^A^4 when selecting window no. 4.
 Use the empty bind command (as in \*Qbind '^\e'\*U) to remove a key binding.
 .sp
 .ne 3
@@ -2470,8 +2482,8 @@ the 
 window (using register p, where a copy remains):
 .PP
 .nf
-	C-a : readreg p /etc/passwd
-	C-a : paste p
+	^A: readreg p /etc/passwd
+	^A: paste p
 .fi
 .sp
 .ne 3
@@ -2571,7 +2583,7 @@ See also chapter \*QWINDOW TYPES\*U.
 .PP
 Set the size of the scrollback buffer for the current windows to \fInum\fP 
 lines. The default scrollback is 100 lines.
-See also the \*Qdefscrollback\*U command and use \*QC-a i\*U to view the 
+See also the \*Qdefscrollback\*U command and use \*Q^Ai\*U to view the 
 current setting.
 .sp
 .ne 3
@@ -2632,7 +2644,7 @@ a '-' character, the shell will be start
 .B "shelltitle \fItitle\fP"
 .PP
 Set the title for all shells created during startup or by
-the C-A C-c command.
+the ^A^C command.
 For details about what a title is, see the discussion
 entitled \*QTITLES (naming windows)\*U.
 .sp
@@ -2664,7 +2676,7 @@ It may be used to give users a chance to
 .B "slowpaste \fImsec\fP"
 .PP
 Define the speed at which text is inserted into the current window by the 
-paste ("C-a ]") command. 
+paste ("^A]") command. 
 If the slowpaste value is nonzero text is written character by character.
 .I screen
 will make a pause of \fImsec\fP milliseconds after each single character write 
@@ -3414,11 +3426,11 @@ were set before the last change was made
 color-change stack).
 .PP
 Examples:
-.IP "\*QG\*U"
+.IP "%{G}"
 set color to bright green
-.IP "\*Q+b r\*U"
+.IP "%{+b r}"
 use bold red
-.IP "\*Q= yd\*U"
+.IP "%{= yd}"
 clear all attributes, write in default color on yellow background.
 .IP "%-Lw%{= BW}%50>%n%f* %t%{-}%+Lw%<"
 The available windows centered at the current window and truncated to
@@ -3445,9 +3457,9 @@ pause the output of the current window.
 You can still send these characters to the current program, but you must use
 the appropriate two-character
 .I screen
-commands (typically \*QC-a q\*U (xon)
-and \*QC-a s\*U (xoff)).
-The xon/xoff commands are also useful for typing C-s and C-q past a terminal
+commands (typically \*Q^Aq\*U (xon)
+and \*Q^As\*U (xoff)).
+The xon/xoff commands are also useful for typing ^S and ^Q past a terminal
 that intercepts these characters.
 .PP
 Each window has an initial flow-control value set with either the
@@ -3455,7 +3467,7 @@ Each window has an initial flow-control 
 option or the \*Qdefflow\*U .screenrc command. Per default the windows
 are set to automatic flow-switching.
 It can then be toggled between the three states 'fixed on', 'fixed off'
-and 'automatic' interactively with the \*Qflow\*U command bound to "C-a f".
+and 'automatic' interactively with the \*Qflow\*U command bound to "^Af".
 .PP
 The automatic flow-switching mode deals with
 flow control using the TIOCPKT mode (like \*Qrlogin\*U does). If
@@ -3467,7 +3479,7 @@ keypad \- when it is enabled, flow-contr
 Of course, you can still manipulate flow-control manually when needed.
 .PP
 If you're running with flow-control enabled and find that pressing the
-interrupt key (usually C-c) does not interrupt the display until another
+interrupt key (usually ^C) does not interrupt the display until another
 6-8 lines have scrolled by, try running
 .I screen
 with the \*Qinterrupt\*U
@@ -3482,21 +3494,21 @@ One disadvantage is that the virtual ter
 non-flushed version of the output, which in rare cases can cause
 minor inaccuracies in the output.
 For example, if you switch screens and return, or update the screen
-with \*QC-a l\*U you would see the version of the output you would
+with \*Q^Al\*U you would see the version of the output you would
 have gotten without \*Qinterrupt\*U being on.
 Also, you might need to turn off flow-control (or use auto-flow mode to turn
 it off automatically) when running a program that expects you to type the
 interrupt character as input, as it is possible to interrupt
 the output of the virtual terminal to your physical terminal when flow-control
 is enabled.
-If this happens, a simple refresh of the screen with \*QC-a l\*U will
+If this happens, a simple refresh of the screen with \*Q^Al\*U will
 restore it.
 Give each mode a try, and use whichever mode you find more comfortable.
 
 
 .SH "TITLES (naming windows)"
 You can customize each window's name in the window display (viewed with the
-\*Qwindows\*U command (C-a w)) by setting it with one of
+\*Qwindows\*U command (^Aw)) by setting it with one of
 the title commands.
 Normally the name displayed is the actual command name of the program
 created in the window.
@@ -3510,7 +3522,7 @@ a \*Qscreen\*U command and thus can have
 .B \-t
 option.
 Interactively, there is the title-string escape-sequence
-(<esc>k\fIname\fP<esc>\e) and the \*Qtitle\*U command (C-a A).
+(<esc>k\fIname\fP<esc>\e) and the \*Qtitle\*U command (^AA).
 The former can be output from an application to control the window's name
 under software control, and the latter will prompt for a name when typed.
 You can also bind pre-defined names to keys with the \*Qtitle\*U command
@@ -3574,7 +3586,7 @@ running, and revert to \*Qcsh\*U upon co
 bind R screen -t '% |root:' su
 .PP
 Having this command in your .screenrc would bind the key
-sequence \*QC-a R\*U to the \*Qsu\*U command and give it an
+sequence \*Q^AR\*U to the \*Qsu\*U command and give it an
 auto-title name of \*Qroot:\*U.
 For this auto-title to work, the screen could look something
 like this:
@@ -3596,10 +3608,10 @@ of the command, and revert to simply \*Q
 .fi
 .sp
 The first binding doesn't have any arguments, so it would prompt you
-for a title. when you type \*QC-a o\*U.
-The second binding would clear an auto-title's current setting (C-a E).
+for a title. when you type \*Q^Ao\*U.
+The second binding would clear an auto-title's current setting (^AE).
 The third binding would set the current window's title to \*Q(unknown)\*U
-(C-a u).
+(^Au).
 .PP
 One thing to keep in mind when adding a null title-escape-sequence to
 your prompt is that some shells (like the csh) count all the non-control
@@ -4416,12 +4428,12 @@ Examples in the 
 .I screen
 distribution package for private and global initialization files.
 .IP $SYSSCREENRC 
-.IP /usr/local/etc/screenrc
+.IP @@SYSCONFDIR@@/screenrc
 .I screen
 initialization commands
 .IP $SCREENRC
 .IP $HOME/.screenrc
-Read in after /usr/local/etc/screenrc
+Read in after @@SYSCONFDIR@@/screenrc
 .IP $SCREENDIR/S-<login>
 .IP /local/screens/S-<login>
 Socket directories (default)
@@ -4442,7 +4454,7 @@ Output log files created by the log func
 or
 .IP /etc/termcap
 Terminal capability databases
-.IP /etc/utmp
+.IP /var/run/utmp
 Login records
 .IP $LOCKPRG
 Program that locks a terminal.
@@ -4555,9 +4567,9 @@ does not make use of hardware tabs.
 must be installed as set-uid with owner root on most systems in order
 to be able to correctly change the owner of the tty device file for
 each window.
-Special permission may also be required to write the file \*Q/etc/utmp\*U.
+Special permission may also be required to write the file \*Q/var/run/utmp\*U.
 .IP \(bu
-Entries in \*Q/etc/utmp\*U are not removed when
+Entries in \*Q/var/run/utmp\*U are not removed when
 .I screen
 is killed with SIGKILL.
 This will cause some programs (like "w" or "rwho")
